tests/nnstreamer_decoder/runTest.sh

```diff
@@ -59,22 +59,36 @@ callCompareTest testcase3.con.log testcase3.dec.log 3 "Compare for case 3" 0 0
 gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc ! tensor_converter ! tensor_decoder mode=direct_video ! video/mpeg ! fakesink" 4F_n 0 1 $PERFORMANCE
 
 # Dynamic tensor config update test
-## After converting the flatbuf to the tensor, it is tested whether the config is updated in runtime.
-gstTest "--gst-plugin-path=${PATH_TO_PLUGIN} videotestsrc num-buffers=3 ! videoconvert ! video/x-raw, width=640, height=480, framerate=5/1,format=RGB ! \
-    tee name=t ! queue ! multifilesink location=\"test_05_raw_%1d.log\"
-    t. ! queue ! tensor_converter ! tensor_decoder mode=flatbuf ! other/flatbuf-tensor ! tensor_converter ! \
-        tensor_decoder mode=direct_video ! multifilesink location=\"test_05_decoded_%1d.log\" sync=true" 5 "Flatbuf dynamic tensor config update" 0 0 $PERFORMANCE
-callCompareTest test_05_raw_0.log test_05_decoded_0.log 5-1 "Compare for case 5-1" 1 0
-callCompareTest test_05_raw_1.log test_05_decoded_1.log 5-2 "Compare for case 5-2" 1 0
-callCompareTest test_05_raw_2.log test_05_decoded_2.log 5-3 "Compare for case 5-3" 1 0
+# Check existence of tensor_decoder flatbuf sub-plugin
+check=$(gst-inspect-1.0 --gst-plugin-path=${PATH_TO_PLUGIN} tensor_decoder | grep flatbuf)
```

Comment:
This is a part of the decoder test, but wouldn't it be better to check the converter sub-plugin together?

Reply:
I updated this commit, please check.